Assets/_BForBoss/_Character/Scripts/PlayerWallRunBehaviour.cs

```diff
@@ -0,0 +1,232 @@
+using ECM2.Characters;
+using ECM2.Components;
+using ECM2.Common;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
```

Comment:
Is this being used somewhere?

Reply:
Good catch, I had that in for debug stuff and forgot about it